.github/workflows/rust.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install libudev
-      run: sudo apt-get install -y libudev-dev
+      run: sudo apt-get update && sudo apt-get install -y libudev-dev
```

Comment:
We need a build variant that ensures that the library can build with `serde` enabled, correct?

Reply:
We also need a matrix that tests we can build no_std and no alloc, for now I think we can survive with just testing the default case and we'll add that later.